receiver/hostmetricsreceiver/example_config.yaml

```diff
@@ -12,6 +12,7 @@ receivers:
       disk:
       filesystem:
       network:
+      virtualmemory:
```

Comment:
This is process... Why virtualmem? Forgot to add process?

Reply:
I forgot to add `virtualmemory` before (thus adding it now 🤦‍♂️),

I intentionally did not add `process` to the default/example config, as process metrics are more expensive to compute and may often not be needed